index.html

```diff
@@ -25,13 +26,153 @@ <h1>I’m Patrick
 			<li><a href=""><img src="/img/medium.svg" alt=""></a></li>
 			<li><a href=""><img src="/img/twiter.svg" alt=""></a></li>
 			<li><a href=""><img src="img/chill.svg" alt=""></a></li>
 		</ul>
 	</section>
+
+	<section class="work">
+		<div>
+			<img src="/img/tonic1.svg" alt="">
+			<h2>TONIC</h2>
+			<ul class="work-exp">
+				<li>Canopy</li>
+				<li><img src="/img/circle.svg" alt=""></li>
+				<li>Back End Dev</li>
+				<li><img src="/img/circle.svg" alt=""></li>
+				<li>2015</li>
+			</ul>
+			<p>A daily selection of privately personalized reads; 
+				no accounts or sign-ups required.</p>
+      <ul class="lang">
+				<li><a href="">html</a></li>
+				<li><a href="">css</a></li>
+				<li><a href="">javascript</a></li>
+			</ul>
+
+			<button class="btn">See Project</button>
+		</div>
+
+		<div>
+			<img src="/img/multistories1.svg" alt="">
+			<h2>Multi-Post
+				Stories</h2>
+			<ul class="work-exp">
+				<li>Canopy</li>
+				<li><img src="/img/circle.svg" alt=""></li>
+				<li>Back End Dev</li>
+				<li><img src="/img/circle.svg" alt=""></li>
+				<li>2015</li>
+			</ul>
+			<p>A daily selection of privately personalized reads; 
+				no accounts or sign-ups required.</p>
+      <ul class="lang">
+				<li><a href="">html</a></li>
+				<li><a href="">css</a></li>
+				<li><a href="">javascript</a></li>
+			</ul>
+
+			<button class="btn">See Project</button>
+		</div>
+
+		<div>
+			<img src="/img/tonic2.svg" alt="">
+			<h2>TONIC</h2>
+			<ul class="work-exp">
+				<li>Canopy</li>
+				<li><img src="/img/circle.svg" alt=""></li>
+				<li>Back End Dev</li>
+				<li><img src="/img/circle.svg" alt=""></li>
+				<li>2015</li>
+			</ul>
+			<p>A daily selection of privately personalized reads; 
+				no accounts or sign-ups required.</p>
+      <ul class="lang">
+				<li><a href="">html</a></li>
+				<li><a href="">css</a></li>
+				<li><a href="">javascript</a></li>
+			</ul>
+
+			<button class="btn">See Project</button>
+		</div>
+
+		<div>
+			<img src="/img/multistories2.svg" alt="">
+			<h2>Multi-Post
+				Stories</h2>
+			<ul class="work-exp">
+				<li>Canopy</li>
+				<li><img src="/img/circle.svg" alt=""></li>
+				<li>Back End Dev</li>
+				<li><img src="/img/circle.svg" alt=""></li>
+				<li>2015</li>
+			</ul>
+			<p>A daily selection of privately personalized reads; 
+				no accounts or sign-ups required.</p>
+      <ul class="lang">
+				<li><a href="">html</a></li>
+				<li><a href="">css</a></li>
+				<li><a href="">javascript</a></li>
+			</ul>
+
+			<button class="btn">See Project</button>
+		</div>
+	</section>
+
+	<section class="about">
+		<div class="about_content">
+			<h2>About Myself</h2>
+			<p>
+				Hello I’m a software developer! 
+				I can help you build a product ,
+				feature or website Look through
+				some of my work and experience!
+				If you like what you see and have a project you need coded, 
+				don’t hestiate to contact me.
+			</p>
+			<a href="">LET’S CONNECT</a>
```

Comment:
Well done on the design, it looks fire 🔥 but talking about semantics, we can improve this a bit more. The title LET'S CONNECT is very important in your portfolio and it needs to be better referenced by browsers 🌐 so in order to achieve that consider using a header tag `hx` so you could have something like below:
 ```
 <h3>LET’S CONNECT</h3>
 ```
 I suppose that also means editing your CSS 🤔  so make sure to edit it or you can also do the following:
 ```
  <h3><a href="">LET’S CONNECT</a></h3>
  ```
This implies of course that you have no styles predefined for `h3` tags, I hope this helps 😉